Gemfile.lock

```diff
@@ -8,115 +8,146 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actionmailer (4.2.1)
-      actionpack (= 4.2.1)
-      actionview (= 4.2.1)
-      activejob (= 4.2.1)
+    actioncable (6.0.2.1)
```

Comment:
Did you just run a general `bundle update`? Is there any danger in upgrading any of these?

Reply:
Yes, guide should handle the dependencies ok, but I think it's safer if we just update the problem gem `nokogiri` in this case. Making that change